types/src/v0/impls/header.rs

```diff
@@ -1119,7 +1171,7 @@ impl BlockHeader<SeqTypes> for Header {
                 // Start with the reward Merkle tree root digest as the base input
                 let digest = header.reward_merkle_tree_root.digest();
                 hasher.update(digest.0);
-                hasher.update(placeholder_1);
+                hasher.update(next_stake_table_hash);
```

Comment:
@bfish713 @imabdulbasit @alxiong do we need this?

The auth root is for data that we need to authenticate on L1 (because we used non keccak hash for the block merkle tree, which is too expensive in the EVM).

Can't a validator complete a block merkle proof and authenticate that against the state of the LC contract?